pyproject.toml

```diff
@@ -43,12 +43,14 @@ retry = "^0.9.2"
 sqlalchemy = "^1.3.3"
 marshmallow-sqlalchemy = "0.24.2"
 storageclient = {git = "https://github.com/uc-cdis/storage-client", rev = "1.0.1"}
-
-# userportaldatamodel = {git = "https://github.com/chicagopcdc/userPortalDataModel.git", tag = "beta"}
```

Comment:
for testing purposes defaulting to a branch is easier to work with, but in prod we want to use the tag since it won't change if someone changes the code in the branch

Reply:
We had recent changes to userportaldatamodel that were needed by amanuensis, should we re-tag the branch to include those changes?

Reply:
We can use the branch for dev since it avoid us to retag every time there is a change. I was just saying to leave that line commented out as a comment instead of removing because when we are doing the new release each month we will use the tag in the master branch instead of the branch. In the time between releases using the branch saved some dev time.

Reply:
userportaldatamodel was retagged as 0.1.0 and updated in poetry, the branch ref was included, but commented out